OOBE Initial

sirredbeard opened the pull request:
This implements the basics of an OOBE experience, building on the previous.

It drops some packages from the default install and moves them into optional packages.

It also incorporates some experimental package scripts like fzf.

The bash logic is still funky and needs work, if you could help me clean it up that would be great.

sirredbeard commented on create-targz-x64.sh:
```diff
@@ -13,7 +13,7 @@ DIST="stable"
 
 # bootstrap image
-sudo cdebootstrap -a $ARCH --include=sudo,locales,git,ssh,apt-transport-https,wget,ca-certificates,man,less,xclip,gnome-themes-standard,gtk2-engines-murrine,dbus-x11,zsh,neovim,curl $DIST $DIST http://deb.debian.org/debian
+sudo cdebootstrap -a $ARCH --include=sudo,locales,git,ssh,apt-transport-https,wget,ca-certificates,man,less,xclip,curl $DIST $DIST http://deb.debian.org/debian
```
Agreed.

sirredbeard commented on linux_files/helpme:
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+explorer.exe https://github.com/WhitewaterFoundry/WLinux
```
Agreed.

sirredbeard commented:
I really need help on this with the logic and handling of the interactive prompts and making sure the command arg is passed and properly handled when script is re-run as an update.

In the long run we will probably move the individual package scripts into their own folder and files, but for now we will keep it monolithic.

patrick330602 commented:
I agree. I am currently trying to make something similar to `yaourt`, that let users to install with ease and let developers to build debian packages with ease. it is still in early stage, but I think in the future this would be a better idea to install some applications without deb packages.

sirredbeard commented:
I like the idea of adopting a consistent non-deb package system for packages not in an apt or deb, as opposed to our own scripts.

We should look more into that after we get bugs out of base monolithic OOBE and in place that at least duplicates our current package sets.

patrick330602 commented:
I don't think international welcome should be added. Windows Console still have issue displaying CJK characters or characters from non-European countries, even it is using UTF-8 code page.
![image](https://user-images.githubusercontent.com/15316889/46023143-93eca180-c116-11e8-8d23-51362cb296de.png)

sirredbeard commented:
The y/n logic in setup is still funky, sometimes I say y it goes n, vice-versa.

I added powershell and filled in some other packages.

I also added pulseaudio support, for audio on WSL.

I apoligize if the code is messy, I just had to get it out of my head before bed.
